Assets/MixedRealityToolkit.Services/InputSystem/RaycastProvider.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.MixedReality.Toolkit.Physics;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class RaycastProvider : BaseDataProvider, IMixedRealityRaycastProvider
```

Comment:
I'd rather call this `DefaultRaycastProvider` to make it clear this is the default implementation.